src/Worker/Grpc/GrpcDurableTaskWorker.Processor.cs

```diff
@@ -186,10 +184,16 @@ async ValueTask<OrchestrationRuntimeState> BuildRuntimeStateAsync(
 
         async Task ProcessWorkItemsAsync(AsyncServerStreamingCall<P.WorkItem> stream, CancellationToken cancellation)
         {
+            // Create a new token source for timing out and a final token source that keys off of them both.
```

Comment:
This comment explains what the code is doing, but can we add comments explaining why we're doing this? It seems a bit unintuitive that we'd want to cancel our long-lived connection after 60 seconds. Only after speaking with Phil and Bernd for a bit did I realize that this is a mechanism for detecting when we've been silently disconnected from the server (because we're not receiving health ping work items).

Reply:
Added comments.